__tests__/app.test.js

```diff
@@ -111,4 +111,40 @@ describe('GET requests', () => {
             })
         });
     });
+
+    describe('Get /api/articles/:article_id/comments', () => {
+        test('200 - responds with array of comments for the given article_id', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.comments).toHaveLength(11)
+                body.comments.forEach((comment) => {
+                    expect(comment).toHaveProperty("comment_id", expect.any(Number))
+                    expect(comment).toHaveProperty("votes", expect.any(Number))
+                    expect(comment).toHaveProperty("created_at", expect.any(String))
+                    expect(comment).toHaveProperty("author", expect.any(String))
+                    expect(comment).toHaveProperty("body", expect.any(String))
+                    expect(comment).toHaveProperty("article_id", expect.any(Number))
+                }) 
+                expect(body.comments).toBeSorted({ key: "created_at", descending: true })
```

Comment:
can we do another 200 that is a correct id but with no comments

Reply:
I've finally done this. Hopefully it's not lost in all the moving and editing!